packages/adapter-next/src/sliceTemplates/AlternateGrid/index.ts

```diff
@@ -0,0 +1,937 @@
+					>
+						{isFilled.image(slice.primary.image) && (
+							<PrismicImage
+								field={slice.primary.image}
+								className={\`
+									es-alternate-grid__image
+									\${
+										slice.variation === "imageRight"
+										? "es-alternate-grid__image--right"
+										: "es-alternate-grid__image--left"
+									}
+								\`}
+							/>
+						)}
+						<div className="es-alternate-grid__primary-content">
+							<div className="es-alternate-grid__primary-content__intro">
+								{isFilled.keyText(slice.primary.eyebrowHeadline) && (
+									<p className="es-alternate-grid__primary-content__intro__eyebrow">
+										{slice.primary.eyebrowHeadline}
+									</p>
+								)}
+								{isFilled.richText(slice.primary.title) && (
+									<div className="es-alternate-grid__primary-content__intro__headline">
+										<PrismicRichText field={slice.primary.title} />
+									</div>
+								)}
+								{isFilled.richText(slice.primary.description) && (
+									<div className="es-alternate-grid__primary-content__intro__description">
+										<PrismicRichText field={slice.primary.description} />
+									</div>
+								)}
+							</div>
+							{slice.items.length > 0 && (
+								<div className="es-alternate-grid__primary-content__items">
+								{slice.items.map((item, i) => (
+									<div key={\`item-\${i + 1}\`} className="es-alternate-grid__item">
+									{isFilled.richText(item.title) && (
+										<div className="es-alternate-grid__item__heading">
+											<PrismicRichText field={item.title} />
+										</div>
+									)}
+									{isFilled.richText(item.description) && (
+										<div className="es-alternate-grid__item__description">
+											<PrismicRichText field={item.description} />
+										</div>
+									)}
+									</div>
+								))}
+								</div>
+							)}
+							</div>
+						</div>
+					
+						<style>
+							{\`
+								.es-bounded {
+									margin: 0px;
+									min-width: 0px;
+									position: relative;
+									padding: 8vw 1.25rem;
+								}
+								.es-bounded-content {
+									min-width: 0px;
+									max-width: 90%;
+									margin: 0px auto;
+								}
+								.es-alternate-grid {
+									font-family: system-ui, sans-serif;
+									background-color: #fff;
+									color: #333;
+								}
+								
+								.es-alternate-grid__content {
+									display: grid;
+									gap: 1.5rem;
+									grid-auto-flow: dense;
+								}
+								
+								.es-alternate-grid__content--with-image {
+									grid-template-columns: 2fr 3fr;
+									gap: 4rem;
+								}
+								
+								@media (min-width: 640px) {
+								.es-alternate-grid__content--with-image {
+									grid-template-columns: repeat(2, 1fr);
+								}
+								}
+								
+								@media (min-width: 1200px) {
+								.es-alternate-grid__content--with-image {
+									grid-template-columns: repeat(2, 1fr);
+								}
+								}
+								
+								.es-alternate-grid__image {
```

Comment:
If we don't have a width height = auto, then the component don't look goods when you don't have tailwinds default that does that for you